odoo/addons/openupgrade_records/models/generate_records_wizard.py

```diff
@@ -17,6 +17,15 @@ class GenerateWizard(models.TransientModel):
     state = fields.Selection(
         [('init', 'init'), ('ready', 'ready')], default='init')
 
+    def quirk_standard_calendar_attendances(self):
+        """ Introduced in Odoo 13. The reinstallation causes a one2many value
```

Comment:
Have you notified odoo of this thing?

Reply:
No, I haven't. I don't think reinstallation of modules is officially supported.